controllers/API/announcement.js

```diff
@@ -15,28 +16,36 @@ router.put('/API/announcement', data.isAuthorised("ALTER_ANNOUNCEMENTS"), functi
 			description: req.param('description')
 		}
 	}, function (err, user) {
-		if (err) throw err;
-		res.send(200);
+		if (err) {
+			logger.warning('Unable to edit announcement.\n' + err);
+			res.send(400);
```

Comment:
According to standard convention res code 400 is indeed more appropriate for error responses am going to change mine as well and update the pdf Charles send to reflect this.